src/Docker/Runner.php

```diff
@@ -207,14 +207,16 @@ private function shouldStoreState($componentId, $configurationId)
      * @param string $mode
      * @param string $jobId
      * @param UsageFileInterface $usageFile
-     * @return Output
+     * @param Output[] $outputs
      */
-    private function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId, UsageFileInterface $usageFile)
+    private function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId, UsageFileInterface $usageFile, array &$outputs)
     {
         $this->loggersService->getLog()->notice(
             "Using configuration id: " . $jobDefinition->getConfigId() . ' version:' . $jobDefinition->getConfigVersion()
             . ", row id: " . $jobDefinition->getRowId() . ", state: " . json_encode($jobDefinition->getState())
         );
+        $outputs[] = new Output();
+        $outputs[count($outputs)-1]->setConfigVersion($jobDefinition->getConfigVersion());
```

Comment:
co si tu misto opakovanyho `$outputs[count($outputs)-1]` udelat promennou `$currentOutput`?

Reply:
Jinak koukam, ze tohle je jediny misto, kde se `Output` vytvari a rovnou v zapeti volas `setConfigVersion`. Proc to neudelat pres konstruktor?

Reply:
$outputs[count($outputs)-1] jsem tam dal, protože jsem si chtěl tak nějak zdůraznit, že opravdu zapisuju do výstupu, takže proměnnou $currentOutput si asi udělat můžu (i když nevím jestli je to dobrej název)
configVersion se sice do construktoru dát dá, ale přišlo mě to jako dost random - není to ani povinný, ani nejdůležitješí, je to jen údaj, kterej je náhodou první k dispozici.

Reply:
Nazev promenny mi je asi burt :) ale ten pristup pres vypocitavany index je proste divny a blbe se to cte.

Co se tyce toho konstruktoru - ok, jen me to tak trklo do oka, jak to jde hned za sebou.